File: samples/Kawa.Sample.Mixed.UseCases/CreateUserResponse.fs
namespace Kawa.Sample.Mixed.UseCases

/// <summary>Carries the mixed sample create-user response.</summary>
/// <param name="id">The generated sample user identifier.</param>
/// <param name="name">The created user name.</param>
type CreateUserResponse(id: string, name: string) =
    /// <summary>Gets the generated sample user identifier.</summary>
    member _.Id = id

    /// <summary>Gets the created user name.</summary>
    member _.Name = name

File: samples/Kawa.Sample.VB.UseCases/CreateUserResponse.vb
Namespace Kawa.Sample.VB.UseCases
    ''' <summary>Carries the VB.NET sample create-user response.</summary>
    Public NotInheritable Class CreateUserResponse
        ''' <summary>Initializes a new instance of the <see cref="CreateUserResponse" /> class.</summary>
        Public Sub New()
        End Sub

        ''' <summary>Initializes a new instance of the <see cref="CreateUserResponse" /> class.</summary>
        ''' <param name="id">The generated sample user identifier.</param>
        ''' <param name="name">The created user name.</param>
        Public Sub New(id As String, name As String)
            Me.Id = id
            Me.Name = name
        End Sub

        ''' <summary>Gets or sets the generated sample user identifier.</summary>
        Public Property Id As String = ""

        ''' <summary>Gets or sets the created user name.</summary>
        Public Property Name As String = ""
    End Class
End Namespace
